src/class-fields/initializer-eval-super/cls-expr-fields-indirect-eval.template

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-performeval-rules-in-initializer
+path: language/expressions/class/fields-derived-cls-indirect-
+name: indirect eval
+---*/
+
+A = class {}
+
+var executed = false;
+C = class extends A {
+  x = (0, eval)('executed = true; /*{ initializer }*/;';
+}
+
+assert.throws(/*{ execution error }*/, function() {
```

Comment:
interesting, without generating the files I can't tell if the script handles multiple words